### DISCUSSION TITLE
How to disable dots under Path hyperlink?

### DISCUSSION THREAD

**Discussion by mkanet** [JanDeDobbeleer/oh-my-posh#4030] [General]
Currently, OhMyPosh **always** has dots displayed under Path hyperlink (See below); which makes the Path text harder to read:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/811008/7595dfdf-5e76-4091-bb2c-b65faebe8871)

How can I disable/remove those dots under the path above?  It is not necessary to display those dots since when we move our mouse cursor to the path, the hyperlink is underlined already.

If there isn't an existing setting that can disable these dots, could you please tell me how to modify OhMyPosh so it doesn't display these dots?  Also, maybe you could add a setting to disable this if you don't have one already.

Thank you for all your development efforts.  OhMyPosh is one of my favorite things in PowerShell!

**Comment by JanDeDobbeleer** [comment]
@mkanet the dots are added by the terminal, not us. We simply tell this is a hyperlink and the terminal renders it according to their preference. The only thing you can do is remove the hyperlink by adjusting the configuration. Usually this is the `{{ path .Path .Location }}` part in the `path` segments `template`. Replace that with `{{ .Path }}` to only have the text.

**Reply by mkanet** [reply]
Thank you for explaining.  I still need to use the Hyperlink; so, I'll just have to tolerate it looking like that.

**Reply by brentsmith-dev** [reply]
For anyone looking to do this in iterm2 they added a "remove hyperlink" option a few years ago which does the trick.
https://gitlab.com/gnachman/iterm2/-/issues/7969
